### ISSUE TITLE
Make Teleport log its version upon service start

### ISSUE THREAD

**Issue by jkendzorra** [gravitational/teleport#3145]
**What happened**: Teleport never emits its own version when it starts

**What you expected to happen**: Logging a line similar to this:
Nov 11 11:08:32 host /usr/bin/teleport[3573]: INFO [PROC:1]    Teleport is starting (Teleport Enterprise v4.1.4git:v4.1.4-0-gc487a75c go1.13.2)
While it's possible to run `teleport version` to get information about the current binary's version, it's not necessarily clear what version is currently running (e.g. upon in-place upgrade).

**Environment**:
- Teleport version (use `teleport version`): any

**Comment by jddonz** [comment]
How about something like this?
```
[AUTH]    Auth service v3.2.14-0-g54ba11ca-dirty is starting on 0.0.0.0:3025.
[PROXY]   Reverse tunnel service v3.2.14-0-g54ba11ca-dirty is starting on 0.0.0.0:3024.
[NODE]    Service v3.2.14-0-g54ba11ca-dirty is starting on 0.0.0.0:3022.
```
or do you want both `Gitref` and `Version` included in the output?

**Comment by fspmarshall** [comment]
Fixed in #3168, thanks @jddonz!
